dashboard/app/models/script_level.rb

```diff
@@ -66,7 +66,8 @@ def oldest_active_level
 
   def active?(level)
     properties_hash = JSON.parse(properties)
-    !properties_hash[level.name] || properties_hash[level.name]['active'] != false
+    variants = properties_hash['variants']
+    !variants || !variants[level.name] || variants[level.name]['active'] != false
```

Comment:
I'm going to avoid making more changes than necessary here given that what we have works, and it's not code that I'm super familiar with.